src/commands/force/source/deploy.ts

```diff
@@ -160,9 +163,11 @@ export class Deploy extends DeployCommand {
     junit: Flags.boolean({ summary: messages.getMessage('flags.junit.summary') }),
   };
   protected readonly lifecycleEventNames = ['predeploy', 'postdeploy'];
-  protected tracking: SourceTracking;
+  protected tracking!: SourceTracking;
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
if this is required to avoid `flags` being possible undefined you could use the non-null assertion:
```diff
-private flags: Interfaces.InferredFlags<typeof Deploy.flags>;
+private flags!: Interfaces.InferredFlags<typeof Deploy.flags>;
````